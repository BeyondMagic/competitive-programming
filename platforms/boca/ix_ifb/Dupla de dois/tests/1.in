4
Daniel 20
Edson 80
Jeremias 80
Guilherme 20
