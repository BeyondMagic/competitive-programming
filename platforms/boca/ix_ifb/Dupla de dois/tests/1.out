0
Guilherme 20 Edson 80
Daniel 20 Jeremias 80
